xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAE.cpp

```diff
@@ -2494,6 +2494,12 @@ bool CActiveAE::HasDSP()
   return m_stats.HasDSP();
 };
 
+bool CActiveAE::GetCurrentSinkFormat(AEAudioFormat &SinkFormat)
+{
+  SinkFormat = m_sinkFormat;
```

Comment:
Ok, then how can I get m_sinkFormat? I really need this to know the speaker layout.

A copy of the data is enough.